packages/grafana-data/src/transformations/fieldReducer.ts

```diff
@@ -313,6 +312,8 @@ export function doStandardCalcs(field: Field, ignoreNulls: boolean, nullAsZero:
       }
     }
 
+    calcs.count++;
```

Comment:
this slightly changes the behavior in that it will also count `undefined` since the early exit above it is for strict `=== null` but the previous version was a `== null` check. i think it's fine at this stage in the pipeline since we don't expect `undefined` values to come from datasources.

Reply:
@leeoniya won't that impact the aligned frame that inserts undefined?

Reply:
these are just calcs, so no? also doesnt all this run before it hits the panel and we don't expect `undefined` to appear before this (coming from datasources)?

Reply:
> does this run before or after transformers?

it runs the first time someone asks for the value, then caches the result in `state` -- this is why transformers need to clear state when futzing with the contents of a field

Reply:
@ryantxu @leeoniya  

I changed the logic here so that null and undefined are treated the same (in regards to count, / ignoreNulls). Like before 

https://github.com/grafana/grafana/pull/67881/commits/7c08ac6308bde0a5033efd05488889f93204e7e6